src/videojs-hls.js

```diff
@@ -279,6 +291,11 @@ videojs.Hls.prototype.play = function() {
     this.mediaIndex = 0;
   }
 
+  if (this.playlists.media() && !this.el().classList.contains('vjs-has-started')) {
```

Comment:
For consistency with the rest of video.js, I think it's still better to use hasClass(). Plus, it's actually less typing here.